api-docs/api-reference/methods/roles/get-list-effective-roles-assigned-to-a-user-v2.0.rst

```diff
@@ -0,0 +1,400 @@
+                          "t1"
+                       ]
+                  },
+                  {
+                       "sourceType": "SYSTEM",
+                       "sourceId": "IDENTITY",
+                       "assignmentType": "TENANT",
+                       "forTenants": [
+                          "t2"
+                       ]
+                  }
+                ]
+            }
+        ]
+    }
+ }
+
+**Across Domains Assignment Example Response**
+
+Assume:
+
+- d1t1 and d1t2 are tenants within the same domain (Domain 1).
+- d2t1 is a tenant in a different domain (Domain 2).
+- The user has the 'observer' role assigned on tenant d1t1, d1t2 in Domain 1
+  and on tenant d2t1 on Domain 2.
+
+JSON::
+
+ {
+    "RAX-AUTH:roleAssignments": {
+        "tenantAssignments": [
+            {
+                "onRole": "8899",
+                "onRoleName": "observer",
+                "forTenants": [
+                     "d1t1",
+                     "d1t2",
+                     "d2t1"
+                 ],
+                "sources": [
+                  {
+                       "sourceType": "USER",
+                       "sourceId": "userId",
+                       "assignmentType": "DOMAIN",
+                       "forTenants": [
+                         "d1t1",
+                         "d1t2",
+                         "d2t1"
+                     ]
+                  }
+                ]
+            }
+        ]
+    }
+ }
+
+**RCN Role Example Response**
+
+Assume:
+
+- The user's RCN contains the domain Domain 1 with the d1t1 and d1t2 tenants.
+- The user's RCN contains the domain Domain 2 with the d2t1 tenants.
+- The RCN role applies to all the mentioned tenants.
+
+
+JSON::
+
+ {
+    "RAX-AUTH:roleAssignments": {
+        "tenantAssignments": [
+            {
+                "onRole": "8899",
+                "onRoleName": "rcn:admin",
+                "forTenants": [
+                   "d1t1",
+                   "d1t2",
+                   "d2t1"
+                ],
+                "sources": [
+                   {
+                      "sourceType": "USER",
+                      "sourceId": "userId",
+                      "assignmentType": "RCN"
+                      "forTenants": [
+                         "d1t1",
+                         "d1t2",
+                         "d2t1"
+                      ]
+                   }
+                ]
+            }
+        ]
+    }
+ }
+
+**User without Tenants Example Response**
+
+A user could be assigned a role that doesn't apply to *any* current tenants for
+the user. For example, the user may have only DOMAIN roles on a domain with no
+tenants. The user could also be assigned an RCN role that doesn't match any
+tenant within the user's RCN. The service returns the role, but shows that that
+source doesn't apply to any tenants.
+
+Assume:
+
+- The user's domain does not contain any tenants
+- The user has the identity:user-admin role
```

Comment:
put ``identity:user-admin`` between double backticks

Also, add a period after the final word (role).